### PR TITLE
Wang_et_al

### PR DESCRIPTION
**Pull request by EFulton1234**

### CODE REVIEW

File: jupyter_book/shared_bibliography/Ellexi/Wang_et_all.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+Paper Links:
+
+
+> []https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0085414&type=printable
```
Comment (jonmatthis) [inline jupyter_book/shared_bibliography/Ellexi/Wang_et_all.md:4]: i like that you put the citation link here

File: jupyter_book/shared_bibliography/Ellexi/Wang_et_all.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+
+### Citation Nature Format:
+  
+1. Wang, Z., Ko, J. H., Challis, J. H. & Newell, K. M. The degrees of freedom problem in human standing posture: collective and component dynamics. PLoS One 9, e85414 (2014).
+
+ 
+### Brief Summary
+
+ 1. What did they do (what activities)?
```
Comment (jonmatthis) [inline jupyter_book/shared_bibliography/Ellexi/Wang_et_all.md:21]: This needs to be updated for this paper

**Review by jonmatthis — CHANGES_REQUESTED** [review]
Please update the content to summary this paper

File: jupyter_book/shared_bibliography/Ellexi/Wang_et_all.md (hunk 1 above)
Comment (madelynredick) [reply jupyter_book/shared_bibliography/Ellexi/Wang_et_all.md:4]: i LOVE that you put the citation link here

Comment (saanvi-vutukur) [reply jupyter_book/shared_bibliography/Ellexi/Wang_et_all.md:4]: so true putting the citation link here enhanced my experience of this paper so much

Comment (glor-n) [reply jupyter_book/shared_bibliography/Ellexi/Wang_et_all.md:4]: this has truly changed my perspective. ill always put the citation link where you did. bibliography icon.

File: jupyter_book/shared_bibliography/Ellexi/Wang_et_all.md (hunk 2 above)
Comment (Robinsbank) [inline jupyter_book/shared_bibliography/Ellexi/Wang_et_all.md:21]: so true 🥳

**Review by Robinsbank — CHANGES_REQUESTED** [review]

File: jupyter_book/shared_bibliography/Ellexi/Wang_et_all.md (hunk 2 above)
Comment (Robinsbank) [reply jupyter_book/shared_bibliography/Ellexi/Wang_et_all.md:21]: h8ter 👁👁

File: jupyter_book/shared_bibliography/Ellexi/Wang_et_all.md [hunk 3]
```diff
@@ -18,37 +17,37 @@ In text citation
...
 
 ### Tags and Keywords
-[what species was studied? What tools were used? What kind of data did they get? What was the main topic?]
 
-#guinea-fowl
 #force-plate
 #kinematic
 #kinetic
 #center-of-mass
+#degree of freedom
```
Comment (jonmatthis) [inline jupyter_book/shared_bibliography/Ellexi/Wang_et_all.md:50]:
```
#degrees-of-freedom
```

**Review by jonmatthis — APPROVED** [review]
this is beautiful 😍

**Review by Robinsbank — APPROVED** [review]

**Review by glor-n — APPROVED** [review]
slayy

**Pull request merged**